src/fmu/dataio/export/rms/volumetrics.py

```diff
@@ -198,61 +139,38 @@ def export_volumetrics(
     project: Any,
     grid_name: str,
     volume_job_name: str,
-    global_config: str | Path | dict = "../../fmuconfig/output/global_variables.yml",
-    forcefolder: str = "",  # unsure if we shall allow this?
-    subfolder: str = "",
-    name: str = "",
-    tagname: str = "",
-    classification: str = "restricted",
-    workflow: str = "rms volumetric run",
-) -> dict[str, str]:
+    config_path: str | Path = "../../fmuconfig/output/global_variables.yml",
+    classification: str | None = None,
```

Comment:
Per discussion:
- Rename this to `export_inplace_volumes()`
- Remove classification, default to restricted